detox/ios/Detox/Actions/NSObject+DetoxActions.m

```diff
@@ -122,6 +124,12 @@ - (void)dtx_longPressAtPoint:(CGPoint)normalizedPoint duration:(NSTimeInterval)d
 	
 	// Add end point
 	[points addObject:@(endPoint)];
+	// This is an ugly hack needed to solve a mystery with a missing last point in the trajectory. We add one more point with the same delta after the end. The OS will happily ignore it.
+	// It is criticul to add a point with the same pace in order to insure it works for all cases. If you use point with a different direction or just (0,0) the OS will not ignore it.
+	CGPoint point = CGPointMake(startPoint.x + (numOfPoints + 1) * xDiffDelta, startPoint.y + (numOfPoints + 1) * yDiffDelta);
+	[points addObject:@(point)];
```

Comment:
hmm, let's test this with different iOS versions (to make sure it works for 15, 16, 17). Can you verify that?